app/controllers/hub/dashboard_controller.rb

```diff
@@ -92,5 +95,33 @@ def self.flatten_filter_options(filter_options, result)
 
     DashboardFilterOption = Struct.new(:value, :model, :children, :has_parent)
 
+    def load_capacity
+      return if @selected.instance_of? Site
+      if @selected.instance_of? Coalition
+        @capacity = @selected.organizations.filter(&:capacity_limit)
+      elsif @selected.instance_of?(Organization) && @selected.capacity_limit
+        @capacity = [@selected]
+      end
+    end
+
+    def capacity_class(organization)
+      if organization.active_client_count > (organization.capacity_limit || 0)
+        "over-capacity"
```

Comment:
should we use i18n here like the rest of the text? Probably doesn't matter much though since so much of the hub is not translated

Reply:
This is used for a css class, so I18n is not needed. I changed the method name to `capacity_css_class` to make this clearer